### PR TITLE
news: add missing newsfile entries

### PR DESCRIPTION
**Pull request by furiel**

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build FAILURE

File: news/bugfix-3083.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+http: fix curl ssl version
```
Comment (alltilla) [inline news/bugfix-3083.md]: I would aim for one-liners, like:
> `http`: Fixed handling of `ssl-version()` option, which was ignored before.

Ultimately, the users don't care about the implementation being with `cURL` or anything else, so I would miss that completely.

My comment is open for arguments :)

**Comment by kira-syslogng** [comment]
Build FAILURE

File: news/bugfix-3083.md (hunk 1 above)
Comment (furiel) [reply news/bugfix-3083.md]: Thanks. I used the oneliner. However, not all options were ignored. So I think it is useful to enumerate those that were ignored.

**Comment by furiel** [comment]
@kira-syslogng retest this please

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**